src/Plugin.NFC.Sample/Plugin.NFC.Sample/MainPage.xaml.cs

```diff
@@ -113,6 +132,7 @@ async void Current_OnMessageReceived(ITagInfo tagInfo)
 				var first = tagInfo.Records[0];
 				await ShowAlert(GetMessage(first), title);
 			}
+			DeviceIsListening = false;
```

Comment:
Are you sure? 
App is still listening after `OnMessageReceived` and/or `OnTagDiscovered` on Android

Reply:
@franckbour from my test I observed the following:
The OnTagDiscovered() is used for the write and clear operations, and the OnMessageReceived() is used for the read operation. After the app performs the read, write or clear operations, it stops listening for NFC tags. So they must press another button so that the app starts listening again, and the user needs to know that the app is listening with a label. Otherwise there is no indication if the problem is with the app or with their unsupported NFC Tags

Reply:
> After the app performs the read, write or clear operations, it stops listening for NFC tags. So they must press another button so that the app starts listening again...

That's weird, I don't reproduce this behavior on my Huawei Mate 10 pro (Android 9) and an old Nexus 5 (Android 6.0.1).

Reply:
Wait for you App is still listening after a message is received and after your perform a write or clear operation on Android? Then how can you make it stop listening?

Reply:
Yes, it is. And it should be on yours too unless you explicitly call `StopListening()`.
Is your phone running Android 10?

Reply:
Yes its running Android 10

Reply:
Ok. So after you scan the first time with your app to “write (text)” and it says that the write operation was successful, when you bring the nfc close to it the second time, does it use the write operation within the app And show it within the app’s dialog? 
Or does it just default to the default behaviour of android phones when NFC is on, where it flips over to using the default read screen? Is that the same behaviour when you delete the app as well?

Reply:
> So after you scan the first time with your app to “write (text)” and it says that the write operation was successful, when you bring the nfc close to it the second time, does it use the write operation within the app And show it within the app’s dialog?

It uses the read operation of the app. I don't need to restart listening for tags with the "Read Tag" Button after write/clear operations.

Reply:
Oh wow, I don't really have a lower Android version device with NFC unfortunately, a ton without NFC though lol

Reply:
OK, my phone has been updated to Android 10 and I reproduce the issue

Reply:
Hmmm so how should we do this ?

Reply:
@saamerm Can you checkout the branch "PR/31"? I did some changes for Android 10. Let me know if it solves the issue!

Reply:
I took a look at it! Your commit on the branch looks pretty good. I didn't get a chance to test it on an NFC device though, that will take me a few days since I'm out. Do you want me to test it anyway?

Reply:
Also do we want to test it on android devices running below Android 10?

Reply:
I tested it in Android 6 and 10. It seems ok but another review will be appreciated!

Reply:
Ok it is some really good and needed work. Let me get home by Wednesday and test it out with my NFC tags